dadi/lib/controller/index.js

```diff
@@ -178,23 +183,25 @@ Controller.prototype.buildInitialViewData = function (req) {
   // add id component from the request
   if (req.params.id) data.id = decodeURIComponent(req.params.id)
 
-  // allow JSON view using ?json=true
-  let json =
-    config.get('allowJsonView') &&
-    urlData.query.json &&
-    urlData.query.json.toString() === 'true'
+  // allow debug view using ?debug=
```

Comment:
Could we keep `json=true` for backwards compatibility, translating it to `debug=json`?